packaging/android/build.sh

```diff
@@ -165,7 +167,7 @@ fi
 if [ ! -e "$PKG_CONFIG_LIBDIR/sqlite3.pc" ] ; then
 	mkdir -p sqlite-build-"$ARCH"
 	pushd sqlite-build-"$ARCH"
-	../sqlite-autoconf-${SQLITE_VERSION}/configure --host=${BUILDCHAIN} --prefix="$PREFIX" --enable-static --disable-shared
+	../sqlite-autoconf-${SQLITE_VERSION}/configure --host=${BUILDCHAIN} --prefix="$PREFIX" --enable-static --disable-shared CFLAGS="-D__ANDROID_API__=$ANDROID_API"
```

Comment:
This was required when I tried r15b.
r15b has a whole lot of other problems so I reverted back to r14b.
These changes don't seem to be required for r14b.